charts/spire/charts/spire-agent/values.yaml

```diff
@@ -215,3 +217,20 @@ extraContainers: []
 initContainers: []
 ## @param hostAliases [array] Customize /etc/hosts file as described here https://kubernetes.io/docs/tasks/network/customize-hosts-file-for-pods/
 hostAliases: []
+
+# NOTE: This is unsupported and only to configure currently supported spire built in plugins but plugins unsupported by the chart.
+# Upgrades wont be tested for anything under this config. If you need this, please let the chart developers know your needs so we
+# can prioritize proper support.
+## @skip unsupportedBuiltInPlugins
+unsupportedBuiltInPlugins:
+  keyManager: {}
+  nodeAttestor: {}
+  svidStore: {}
+  workloadAttestor: {}
+
+## @skip customPlugins
```

Comment:
I know we did the skip for server so it's fine to leave it as is. Just wondering if it would be good to document?